### DISCUSSION TITLE
Is this project alive?

### DISCUSSION THREAD

**Discussion by davidhealey** [socioboard/Socioboard-5.0#418] [General]
Before I look into using this just wanted to check if it's still active?

**Comment by weismannweb** [comment]
+1 i am looking for this and i love its using laravel, i also suggest  animated gif of the app to readme, im installing now to se ehow far along it is

**Comment by sureshbabu1995** [comment]
Hi @davidhealey @weismannweb -- yes, this is still active and we'll be bringing more profiles like blogs, chat support, etc...,

Please try to set it up. We've solved many issues of installations in the closed issues, if still there's an issue, feel free to raise a ticket.

**Reply by bestkiller92** [reply]
Thanks for that reply :p I will try to set it up too

**Reply by AloyASen** [reply]
I have recently tried to set it up..... All went as prescribed, but First time user registration was not going through.... The api.... Was returning unauthorized 

I think the Socioboard system cannot be built or tested in the **localhost** environment.... As was written in a configuration file...... Any ways how to test it in local system @sureshbabu1995

**Comment by TimboGP** [comment]
What about the website socioboard.com?

**Comment by TechnologyClassroom** [comment]
See the issue https://github.com/socioboard/Socioboard-5.0/issues/102 for a summary of where the project is at and recent developments.
